releasenotes/notes/py310-support-869d47583c976eef.yaml

```diff
@@ -0,0 +1,17 @@
+---
+features:
+  - |
+    Added support for running with Python 3.10. This includes publishing
+    precompiled binaries to PyPI for Python 3.10 on supported platforms.
```

Comment:
Nice - in #7102 I didn't mention it because I thought we'd put it in the release prelude, but it's better to have an explicit note for it as well.

Reply:
Yeah, we should still put it in the prelude but I was thinking it'd be good to also list it as a new feature explicitly too.